manager/manager.go

```diff
@@ -308,6 +311,7 @@ func (m *Manager) Run(parent context.Context) error {
 	api.RegisterRaftMembershipServer(m.server, proxyRaftMembershipAPI)
 	api.RegisterControlServer(m.localserver, localProxyControlAPI)
 	api.RegisterControlServer(m.server, authenticatedControlAPI)
+	api.RegisterResourceAllocatorServer(m.server, authenticatedResourceAPI)
```

Comment:
It is automatic if you wrap the service with `NewRaftProxyResourceAllocatorServer`. Notice that the dispatcher has:

```go
proxyDispatcherAPI := api.NewRaftProxyDispatcherServer(authenticatedDispatcherAPI, cs, m.RaftNode, ca.WithMetadataForwardTLSInfo)
```

I would follow the dispatcher example since this service is similar.

Note that the proxy should wrap the authenticator, rather then vice versa. This is so that authorization always happens on the leader.